jobs/multi-arch-pipeline.Jenkinsfile

```diff
@@ -0,0 +1,493 @@
+}
+
+// Share with the Fedora testing account so we can test it afterwards
+FEDORA_AWS_TESTING_USER_ID = "013116697141"
+
+// Base URL through which to download artifacts
+BUILDS_BASE_HTTP_URL = "https://builds.coreos.fedoraproject.org/prod/streams"
+
+def coreos_assembler_image
+if (official) {
+    coreos_assembler_image = "coreos-assembler:main"
+} else {
+    coreos_assembler_image = "${developer_prefix}-coreos-assembler:main"
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             // list devel first so that it's the default choice
+             choices: (streams.development + streams.production + streams.mechanical),
+             description: 'Fedora CoreOS stream to build'),
+      string(name: 'VERSION',
+             description: 'Override default versioning mechanism',
+             defaultValue: '',
+             trim: true),
+      booleanParam(name: 'FORCE',
+                   defaultValue: false,
+                   description: 'Whether to force a rebuild'),
+      booleanParam(name: 'MINIMAL',
+                   defaultValue: (official ? false : true),
+                   description: 'Whether to only build the OSTree and qemu images'),
+      booleanParam(name: 'ALLOW_KOLA_UPGRADE_FAILURE',
+                   defaultValue: false,
+                   description: "Don't error out if upgrade tests fail (temporary)"),
+      // use a string here because passing booleans via `oc start-build -e`
+      // is non-trivial
+      choice(name: 'AWS_REPLICATION',
+             choices: (['false', 'true']),
+             description: 'Force AWS AMI replication for non-production'),
+      string(name: 'COREOS_ASSEMBLER_IMAGE',
+             description: 'Override coreos-assembler image to use',
+             defaultValue: "${coreos_assembler_image}",
+             trim: true),
+      string(name: 'ARCH',
+             description: 'The target architecture',
+             defaultValue: 'aarch64',
+             trim: true),
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '60',
+        artifactNumToKeepStr: '3'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+def is_mechanical = (params.STREAM in streams.mechanical)
+// If we are a mechanical stream then we can pin packages but we
+// don't maintin complete lockfiles so we can't build in strict mode.
+def strict_build_param = is_mechanical ? "" : "--strict"
+
+// Note the supermin VM just uses 2G. The really hungry part is xz, which
+// without lots of memory takes lots of time. For now we just hardcode these
+// here; we can look into making them configurable through the template if
+// developers really need to tweak them (note that in the default minimal devel
+// workflow, only the qemu image is built).
+def cosa_memory_request_mb
+if (official) {
+    cosa_memory_request_mb = 6.5 * 1024
+} else {
+    cosa_memory_request_mb = 2.5 * 1024
+}
+cosa_memory_request_mb = cosa_memory_request_mb as Integer
```

Comment:
Since everything happens on the remote, we don't actually need to allocate a lot of memory here.

Reply:
yeah i can bump it down +1